src/plugins/intel_gpu/src/graph/impls/onednn/gemm_onednn.hpp

```diff
@@ -62,6 +62,11 @@ struct GemmImplementationManager : public ImplementationManager {
             !one_of(out_layout.format.value, supported_formats))
             return false;
 
+        std::vector<size_t> out_transpose_order(std::begin(gemm_prim->output_transpose_order), std::end(gemm_prim->output_transpose_order));
+        if (out_transpose_order == format::traits(static_cast<format::type>(format::bxfy))._order) {
```

Comment:
nit: 
seems this case is not needed:
```suggestion
        if (out_transpose_order == format::traits(format::bxfy)._order) {
```

Reply:
done